casepro/statistics/signals.py

```diff
@@ -52,3 +53,22 @@ def record_incoming_labelling(sender, instance, action, reverse, model, pk_set,
     elif action == 'pre_clear':
         for label in instance.labels.all():
             DailyCount.record_removal(day, DailyCount.TYPE_INCOMING, label)
+
+
+@receiver(post_save, sender=CaseAction)
+def record_new_case_action(sender, instance, created, **kwargs):
+    """
+    This is where we keep track of DailyCounts for users within organisations
+    """
+    org = instance.case.org
+    user = instance.created_by
+    partner = instance.case.assignee
+
+    day = datetime_to_date(instance.created_on, instance.case.org)
+    if instance.action == CaseAction.OPEN:
+        DailyCount.record_item(day, DailyCount.TYPE_CASE_OPENED, org, user)
+        DailyCount.record_item(day, DailyCount.TYPE_CASE_OPENED, partner)
+
+    elif instance.action == CaseAction.CLOSE:
```

Comment:
Maybe we only count if this is the first time this case has been closed? Otherwise users can up their stats by closing/reopening cases.